modules/ROOT/pages/provisioning-nutanix.adoc

```diff
@@ -59,4 +59,96 @@ EOF
 
 == Launching a VM instance
 
-You can provision an FCOS instance using the Nutanix Prism Central web portal or via the Prism Central API with `curl`. Ignition configuration can be passed to the VM as a "cloud-init custom script".
+You can provision an FCOS instance using the Nutanix Prism Central web portal or via the Prism Central API with `curl`. Ignition configuration can be passed to the VM as a "cloud-init custom script". For example, to launch a VM using the API:
+
+[source, bash]
+----
+API_HOST=<Prism Central hostname>
+API_USERNAME=<username>
+API_PASSWORD=<password>
+CLUSTER_REFERENCE_NAME=<name of cluster to use>
+CLUSTER_REFERENCE_UUID=<uuid of cluster to use>
+IGNITION_CONFIG=config.ign
+ENCODED_CONFIG="$(cat ${IGNITION_CONFIG} | base64 -w 0)"
+ENCODED_CREDS="$(echo -n "${API_USERNAME}:${API_PASSWORD}" | base64)"
+IMAGE_ID=$(curl -X POST --header "Content-Type: application/json" \
+               --header "Accept: application/json" \
+               --header "Authorization: Basic ${ENCODED_CREDS}" \
+               "https://${API_HOST}:9440/api/nutanix/v3/images/list"
+               -d '{ "kind": "image","filter": "", "length": 30, "offset": 0}' | \
+               jq -r '.entities[] | select(.spec.name == "${IMAGE_NAME}") | .metadata.uuid')
+SUBNET_REFERENCE_NAME=<name of subnet to use>
+SUBNET_REFERENCE_UUID=<uuid of subnet to use>
```

Comment:
Let's group the user-configured variables together, then add a blank line, then list the computed variables.